a10_octavia/controller/worker/flows/a10_health_monitor_flows.py

```diff
@@ -94,12 +101,15 @@ def get_delete_health_monitor_flow(self):
             requires=constants.POOL))
         delete_hm_flow.add(database_tasks.MarkLBAndListenersActiveInDB(
             requires=[constants.LOADBALANCER, constants.LISTENERS]))
-        delete_hm_flow.add(vthunder_tasks.WriteMemory(
-            name=a10constants.WRITE_MEM_FOR_LOCAL_PARTITION,
+        if CONF.a10_house_keeping.disable_write_memory:
```

Comment:
maybe we can move this check inside WriteMemory Task, and it will be easier to maintain our code.

Reply:
Will shift this check inside WriteMemory Task